src/bin/init-test-db.ts

```diff
@@ -0,0 +1,31 @@
+import { ExecException, exec } from 'child_process';
+import { upgradeDb } from '../filler/upgrade-db';
+import PostgresConnection from '../connections/postgres';
+
+async function main(): Promise<void> {
+    const db = 'atomichub-test';
+    await execCommand(`dropdb --username postgres --force --if-exists ${db}`);
+    await execCommand(`createdb --username postgres ${db}`);
+
+    const database = new PostgresConnection('localhost', 5432, 'postgres', 'x', db);
+
+    await upgradeDb(database);
+
+    process.exit(0);
+}
+
+main().then(console.info, console.error);
```

Comment:
Is there are reason why we dont use winston as logger in this file?

Reply:
No, just didn't think of it